src/core/thread/mle.cpp

```diff
@@ -2101,10 +2108,11 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
     SuccessOrExit(error = Tlv::GetTlv(aMessage, Tlv::kNetworkData, sizeof(networkData), networkData));
 
     // Active Timestamp
-    if (Tlv::GetTlv(aMessage, Tlv::kActiveTimestamp, sizeof(activeTimestamp), activeTimestamp) == kThreadError_None)
-    {
-        VerifyOrExit(activeTimestamp.IsValid(), error = kThreadError_Parse);
+    SuccessOrExit(error = Tlv::GetTlv(aMessage, Tlv::kActiveTimestamp, sizeof(activeTimestamp), activeTimestamp));
```

Comment:
Not sure whether or not MLE Active Timestamp TLV is necessary in ChildIdResponse message. If it's optional, using `SuccessOrExit()` will miss the following check.(i.e. send Address Solicit to request Router ID).